src/react/components/ui/scroll-area.tsx

```
@@ -0,0 +1,57 @@
+import * as ScrollAreaPrimitive from '@radix-ui/react-scroll-area'
+import * as React from 'react'
+
+import { cn } from '@/client/lib/utils/cn'
+
+
+function ScrollArea({
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)